spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerAutoConfiguration.java

```diff
@@ -133,4 +134,12 @@ public SpannerMutationFactory spannerMutationFactory(
 			SpannerMappingContext spannerMappingContext) {
 		return new SpannerMutationFactoryImpl(spannerConverter, spannerMappingContext);
 	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public MappingSchemaOperations mappingSchemaGenerator(
```

Comment:
How about`SpannerSchemaOperations`? Nevertheless, the method name should be synced up with the class name.

Reply:
done